python_modules/dagster/dagster/_core/storage/tags.py

```diff
@@ -68,6 +68,8 @@
     MAX_RUNTIME_SECONDS_TAG,
 ]
 
+GLOBAL_CONCURRENCY_TAG = f"{SYSTEM_TAG_PREFIX}pool"
```

Comment:
is this a locked name?

Reply:
not a locked name... I think I probably prefer having an explicit `concurrency_key` arg on the op definition, rather than using a special tag key.

And I like using `concurrency_key` over `pool`, but it can get confusing with both run-scoped op concurrency and run-level concurrency.

Reply:
`global_concurrency_key` ? what happens if we change this - nothing right since its just how the user process interacts with user definitions so its isolated ?

Reply:
"deployment scoped" or "instance scoped" might be more appropriate names than "global"